packages/field/src/InteractionStateMixin.js

```diff
@@ -70,12 +70,16 @@ export const InteractionStateMixin = dedupeMixin(
         return !!value || isNumberValue || isBooleanValue;
       }
 
+      get _leaveNode() {
```

Comment:
it's probably better to not do this

implementers should capture and refire the leave event on the host instead (also results in a better API for them as well)